app/src/main/java/com/infomaniak/mail/data/LocalSettings.kt

```diff
@@ -204,11 +204,12 @@ class LocalSettings private constructor(context: Context) {
 
         private val DEFAULT_THREAD_DENSITY = ThreadDensity.LARGE
         private val DEFAULT_THEME = Theme.SYSTEM
-        val DEFAULT_ACCENT_COLOR = AccentColor.PINK
+        private val DEFAULT_ACCENT_COLOR = AccentColor.PINK
         private val INITIAL_SWIPE_ACTION = SwipeAction.TUTORIAL
         val DEFAULT_SWIPE_ACTION_RIGHT = SwipeAction.READ_UNREAD
         val DEFAULT_SWIPE_ACTION_LEFT = SwipeAction.DELETE
         private val DEFAULT_EXTERNAL_CONTENT = ExternalContent.ASK_ME
+        private const val DEFAULT_RECENT_SEARCHES = "[]"
```

Comment:
It disappears with the previous remark

Reply:
Related to https://github.com/Infomaniak/android-mail/pull/648#discussion_r1119791175